lib/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set (QUDA_OBJS
   dirac_domain_wall_4d.cpp dirac_mobius.cpp dirac_twisted_clover.cpp
   dirac_twisted_mass.cpp tune.cpp
   llfat_quda.cu gauge_force.cu gauge_random.cu
-  field_strength_tensor.cu clover_quda.cu dslash_quda.cu
+  gauge_field_strength_tensor.cu clover_quda.cu dslash_quda.cu
```

Comment:
Why the change in file name?

Reply:
Exactly. I'll put all the gauge algorithms that use the same instantiation structure in files prepended with `gauge_` like the `dslash_` ones. Then they'll all be contiguous in the GitHub page too.